website/src/content/deployment-examples/google-cloud-storage.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Deploying to Google cloud
+---
+
+This deployment example refers to [Google cloud Static Web Apps](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#0)
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [google](https://accounts.google.com/SignUp) account.
+- A custom domain
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationid) you got when you configured the Custom Application in the Merchant Center.
+
+
+```json title="custom-application-config.json"
+{
+  "env": {
+    "production": {
+      "applicationId": "ckvtahxl90097sys6har1e6n3",
+      "url": "https://<custom_domain>"
+    }
+  }
+}
+```
+
+## Using environment variables
+
+In case you want to avoid hardcoding certain values, for example the Application ID, or the Project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
+
+```json title="custom-application-config.json" highlightLines="4"
+{
+  "env": {
+    "production": {
+      "applicationId": "${env:APPLICATION_ID}",
+      "url": "https://<custom_domain>"
+    }
+  }
+}
+```
```

Comment:
I feel I'm missing a fragment where we configure those environment variables somewhere in Google's infrastructure (like the ones we have in other pages starting with `If your Custom Application config requires environment variables, make sure to provide them in...`). 

And if there is actually no option to do that in case of Cloud Storage, perhaps we should provide an example how to that using e.g. GitHub Actions or consider removing this fragment.

Reply:
Thanks for the changes. It now occurred to me I was wrong, we're documenting a fully manual deployment process. User is supposed to build the app locally using local `.env` file and then manually upload it. As much as integration with GitHub Actions is a nice addition in general, my suggestion didn't make much sense. Sorry for that.

Reply:
Totally fine! Thanks for circling back on this 😁